PBM-1376 Added SSE-S3 encryption description (#201)
percona/pbm-docs: docs/details/storage-configuration.md

  sseCustomerKey: <your_encryption_key>
``` 

##### SSE-S3 encryption

!!! admonition "Version added: [2.6.0](../release-notes/2.6.0.md)" 

> Percona Backup for MongoDB supports server-side encryption with Amazon S3 managed keys (SSE-S3). This is the default encryption method in Amazon AWS. All new objects added to an S3 bucket are automatically encrypted with no impact on performance.

Reviewer: ```suggestion
Percona Backup for MongoDB supports server-side encryption with Amazon S3 managed keys (SSE-S3), the default encryption method in Amazon AWS. All new objects added to an S3 bucket are automatically encrypted without impacting performance.
```

> To use SSE-S3 encryption, specify the following following parameters in the Percona Backup for MongoDB configuration file:   

Reviewer: ```suggestion
To use SSE-S3 encryption, specify the following parameters in the Percona Backup for the MongoDB configuration file:
```
